packages/path_provider/path_provider_windows/README.md

```diff
@@ -0,0 +1,32 @@
+# path_provider_windows
+
+The Windows implementation of [`path_provider`][1].
+
+*This is not an official package, and is not ready for use!*
+
+**Please set your constraint to `path_provider_windows: '>=0.0.y+x <2.0.0'`**
+
+## Backward compatible 1.0.0 version is coming
+The plugin has reached a stable API, we guarantee that version `1.0.0` will be backward compatible with `0.0.y+z`.
+Please use `path_provider_windows: '>=0.0.y+x <2.0.0'` as your dependency constraint to allow a smoother ecosystem migration.
+For more details see: https://github.com/flutter/flutter/wiki/Package-migration-to-1.0.0
+
+## Usage
+
+### Import the package
+
+This package has been endorsed, meaning that you only need to add `path_provider`
+as a dependency in your `pubspec.yaml`. It will be automatically included in your app
+when you depend on `package:path_provider`.
+
+This is what the above means to your `pubspec.yaml`:
+
+```yaml
+...
+dependencies:
+  ...
+  path_provider: ^1.5.1
```

Comment:
path_provider should get a version bump in this PR, and this should refer to that version.